paper.md

```diff
@@ -33,37 +33,41 @@ bibliography: paper.bib
+HeuDiConv is written in Python, and supports the DICOM specification for input parsing, and the BIDS specification for output construction.
+The support for these standards is extensive, and HeuDiConv can handle complex organization scenarios such as arise for specific data types (e.g. multi-echo sequences, or single-band reference volumes).
+In addition to generating valid BIDS outputs, additional support is offered for custom output layouts.
+This is obtained via a set of example heuristiscs, which can be modified into supplemental heuristics expressed as simple Python functions, thus providing full flexibility and maintaining user accessibility.
+HeuDiConv further integrates with DataLad [@datalad], and can automatically prepare DataLad archives with optional obfuscation of sensitive data and metadata, including obfuscating patient visit timestamps in version control.
+As a result, given its extensibility, large modality support, and integration with advanced data management technologies, HeuDiConv has become a mainstay in numerous neuroimaging workflows, and constitutes a powerful and highly adaptable tool of potential interest to large swathes of the neuroimaging community.
+
+
+# Statement of Need
+
+Neuroimaging is an empirical research area which relies heavily on efficient data acquisition, harmonization, and processing.
+Neuroimaging data sourced from medical imaging equipment, and in particular magnetic resonance imaging (MRI) scanners, can be exported in numerous formats, among which DICOM (Digital Imaging and Communications in Medicine) is most prominent.
+DICOM data are often transmitted to PACS (Picture Archiving and Communication Systems) servers for archiving or further processing.
+Unlike in clinical settings, where data are interfaced with directly from PACS in the DICOM format, in neuroimaging research, tools typically require data files in the much simpler (and metadata-restricted) NIfTI [@nifticlib] format.
+Tools such as `dcm2niix` [@Li_2016] can be used to convert *individual* DICOM files into named NIfTI files, and can extract metadata fields not covered by the NIfTI header into sidecar `.json` files.
+However, the scope of such tools is limited, as it does not extend to organizing *multiple* files within a study.
+
+HeuDiConv was developed to provide flexible tooling so that labs may rapidly and consistently convert collections of DICOM files into collections of NIfTI files in customizable file system hierarchies.
+Since the inception of HeuDiConv in 2014, the BIDS standard [@GAC+16] was established.
+This standard formalizes data file hierarchies and metadata storage in a fashion which, due to its community-driven nature, is both highly optimized and widely understood by analysis tools.
+Since then, DICOM conversion to NIfTI files contained within a BIDS hierarchy has emerged as the most frequent use-case for HeuDiConv.
```

Comment:
I think we need to explicitly mention that manual is possible but error prone. It seems to be good somewhere here ;):

```suggestion
As manual renaming of the files and composition of sidecar metadata files for BIDS dataset is tedious and error prone process if done manually, automation tools are needed.
Since then, DICOM conversion to NIfTI files contained within a BIDS hierarchy has emerged as the most frequent use-case for HeuDiConv.
```

Reply:
better inserted here, since otherwise it sounds like it's a specific issue with BIDS. [bd36208](https://github.com/nipy/heudiconv-joss-paper/pull/13/commits/bd36208181b55a274813dd3c3dcc96a86ec920b8)